faunadb/_json.py

```diff
@@ -10,7 +11,10 @@ def parse_json(json_string):
   Parses a JSON string into python values.
   Also parses :any:`Ref`, :any:`Set`, :any:`FaunaTime`, and :class:`date`.
   """
-  return loads(json_string, object_hook=_parse_json_hook)
+  try:
+    return loads(json_string, object_hook=_parse_json_hook)
+  except ValueError:
+    raise InvalidResponse("Bad json: %s" % json_string)
```

Comment:
I'd actually rather see something like "JSON parse error: <ValueError message>" than the current message here, as iirc the `ValueError` from the json parser will explain what it didn't like, and we will already have access to it here.